README.dev.md

```diff
@@ -54,6 +51,42 @@ the rule is intended to fix. Words such as 'forbid' should be omitted. For examp
 name for the rule that warns of overly long lines is `LineTooLong`, and not something
 like `AvoidLineTooLong` or `KeepLinesShort`.
 
+#### Rule testing: fixtures and snapshots
+
+To test rules, Fortitude uses snapshots of Fortitude's output for a given file (fixture). Generally, there
+will be one file per rule (e.g., `E402.py`), and each file will contain all necessary examples of
+both violations and non-violations. `cargo insta review` will generate a snapshot file containing
+Fortitude's output for each fixture, which you can then commit alongside your changes.
+
+Once you've completed the code for the rule itself, you can define tests with the following steps:
+
+1. Add a Python file to `fortitude/resources/test/fixtures/[category]` that contains the code you
```

Comment:
Similarly, should be a Fortran file. There are a few other Python references in this doc.